src/IssueBookDao.java

```diff
@@ -0,0 +1,63 @@
+import java.sql.*;
+public class IssueBookDao {
+	
+public static boolean checkBook(String bookcallno){
+	boolean status=false;
+	try{
+		Connection con=DB.getConnection();
+		PreparedStatement ps=con.prepareStatement("select * from books where callno=?");
+		ps.setString(1,bookcallno);
+	    ResultSet rs=ps.executeQuery();

```

Comment:
Recommendation generated by Amazon CodeGuru Reviewer

**Problem**
This line of code contains a resource that might not be closed properly. Resource leaks can cause your system to slow down or crash.

**Fix**
Consider closing the following resource: *rs*. The resource is referenced in statements at the following lines: 8, 9, 10, 11. The resource closure statement is at line: 12. There are other execution paths that do not contain closure statements, e.g., when exception is thrown by ResultSet.next. Either a) close *rs* in a try-finally block or b) close the resource by declaring *rs* in a try-with-resources block.

**More info**
[View resource management guidelines at oracle.com](https://www.oracle.com/technetwork/articles/java/trywithresources-401775.html) (external link).